vnext/PropertySheets/Bundle.props

```diff
@@ -49,6 +54,27 @@
     <!-- Location where source map file for the JS bundle will be created -->
     <BundleSourceMapDir Condition="'$(BundleSourceMapDir)' == ''">$(OutDir)\sourcemaps\react</BundleSourceMapDir>
 
+    <!-- Base (extensionless) filename for bundle sourcemaps -->
+    <BaseBundleSourceMap Condition="'$(BaseBundleSourceMap)' == ''">$([MSBuild]::MakeRelative($(BundleCommandWorkingDir), $([MSBuild]::NormalizePath('$(BundleSourceMapDir)\$(BundleAssetName)'))))</BaseBundleSourceMap>
+    
+    <!-- Source map file for the bundle (composed from packager + compiler if using Hermes) -->
+    <BundleSourceMap Condition="'$(BundleSourceMap)' == ''">$(BaseBundleSourceMap).map</BundleSourceMap>
+
+    <!-- Filename for source maps generated by metro-->
+    <PackagerSourceMapExtension Condition="'$(PackagerSourceMapExtension)' == '' AND '$(UseHermes)' != 'true'">.map</PackagerSourceMapExtension>
+    <PackagerSourceMapExtension Condition="'$(PackagerSourceMapExtension)' == '' AND '$(UseHermes)' == 'true'">.packager.map</PackagerSourceMapExtension>
+    <PackagerSourceMap Condition="'$(PackagerSourceMap)' == ''">$(BaseBundleSourceMap)$(PackagerSourceMapExtension)</PackagerSourceMap>
+
+    <!-- Filename for source maps generated by Hermes -->
+    <HermesSourceMap Condition="'$(HermesSourceMap)' == '' AND '$(UseHermes)' == 'true'">$(BaseBundleSourceMap).compiler.map</HermesSourceMap>
+
+    <!-- Command to compile Hermes bytecode (using win32 instead of HermesArch because of https://github.com/microsoft/hermes-windows/issues/24) -->
+    <HermesCompilerCommand Condition="'$(HermesCompilerCommand)' == ''">$(HermesPackage)\lib\win32\release\x86\hermes.exe</HermesCompilerCommand>
```

Comment:
>x86 [](start = 107, length = 3)

Is the Hermes bytecode platform-independent? Would it be better to use the x64 compiler even if it was (in case we're dealing with a huge bundle and are bumping against the 4Gb address space limit?

Reply:
The bytecode is platform independent so we should be good. Went with x86 just because there's greater compatibility (e.g. with ARM64 devices).